spec/controllers/v0/profile/ch33_bank_accounts_controller_spec.rb

```diff
@@ -108,7 +108,7 @@ def send_successful_update
       it 'submits the update req and rerenders index' do
         send_successful_update
 
-        expect_find_ch33_dd_eft_res
+        # expect_find_ch33_dd_eft_res
```

Comment:
Is there a reason why this is commented out? I tested locally with it uncommented and it passes.